test/test_seqno.cpp

```diff
@@ -41,6 +41,12 @@ TEST(CSeqNo, seqoff)
     EXPECT_EQ(CSeqNo::seqoff(0x7FFFFFFF, 1),    2);
 }
 
+TEST(CSeqNo, seqlen)
+{
+    EXPECT_EQ(CSeqNo::seqlen(125, 125), 1);
+    EXPECT_EQ(CSeqNo::seqlen(125, 126), 2);
+}
+
```

Comment:
It would be nice to have also a test for this `getFlightSpan()` - something like that you simulate sending 5 packets, then check the value for `getFlightSpan()`, then allow all of them to be ACK-ed, and then check again (when there are expected no unacknowledged packets). It would require some extra mechanism to prevent ACKs from being sent, or possibly turn on losing all packets for the first 5 sending, then unblock to make them retransmitted.